draft-ietf-mls-protocol.md

```diff
@@ -1641,46 +1641,32 @@ When parsing a SenderData struct as part of message decryption, the recipient
 MUST verify that the LeafNodeRef indicated in the `sender` field identifies a
 member of the group.
 
-# Ratchet Tree Operations
-
-The ratchet tree for an epoch describes the membership of a group in that epoch,
-providing public-key encryption (HPKE) keys that can be used to encrypt to subsets of
-the group as well as information to authenticate the members.  In order to
-reflect changes to the membership of the group from one epoch to the next,
-corresponding changes are made to the ratchet tree.  In this section, we
-describe the content of the tree and the required operations.
-
-## Ratchet Tree Node Contents
-
-As discussed in {{ratchet-tree-nodes}}, the nodes of a ratchet tree contain
-several types of data describing individual members (for leaf nodes) or
-subgroups of the group (for parent nodes).  Parent nodes are simpler:
-
-~~~~~
-struct {
-    HPKEPublicKey public_key;
-    opaque parent_hash<V>;
-    uint32 unmerged_leaves<V>;
-} ParentNode;
-~~~~~
-
-The `public_key` field contains a HPKE public key whose private key is held only
-by the members at the leaves among its descendants.  The `parent_hash` field
-contains a hash of this node's parent node, as described in {{parent-hash}}.
-The `unmerged_leaves` field lists the leaves under this parent node that are
-unmerged, according to their indices among all the leaves in the tree.
+# ClientInfo Object
```

Comment:
I would probably keep this under `# Ratchet Tree Node Contents`, even if we're going to rename to `ClientInfo`.  In general, struct are introduced in context, e.g., GroupInfo and Welcome are introduced in the context of the relevant join operations.